WcaOnRails/config/locales/en.yml

```diff
@@ -953,9 +953,9 @@ en:
     registration_requirements: "Registration requirements for the competition:"
     import:
       title: "Import registrations for %{comp}"
-      warning: "This action is not reversible, make sure to do this once the registration is closed."
+      warning: "This action is not reversible and should be done only once, so don't do this until after registration is closed."
```

Comment:
If I'm understanding correctly, doesn't the code only allow this to happen once? If so, then I think we should say "and can only be done once".

However, as mentioned elsewhere, I think we should allow people to re-import. Even if they have to do something like manually delete every registration to free things up for re-importing.